pkg/workflow/v1/validate.go

```diff
@@ -0,0 +1,9 @@
+package v1
+
+import "github.com/puppetlabs/nebula-sdk/pkg/util/typeutil"
+
+// Validates a yaml document according to the schema specification
```

Comment:
This doc should start with `ValidateYAML ...` per the Go documentation conventions (https://blog.golang.org/godoc-documenting-go-code).